packages/flutter_tools/templates/create/.analysis_options.tmpl

```diff
@@ -0,0 +1,70 @@
+# This file contains the analysis options used by the Dart analyzer
+# to decide what errors, warnings, and hints should be shown.
+# See the configuration guide for more
+# https://github.com/dart-lang/sdk/tree/master/pkg/analyzer#configuring-the-analyzer
+
+analyzer:
+  language:
+    enableStrictCallChecks: true
+    enableSuperMixins: true
+  strong-mode: false
```

Comment:
My understanding is that in Dart 2.0 there is only strong mode. All internal Dart code is strong mode clean or soon will be, and we ourselves enforce strong mode on the flutter repo. Strong mode catches more bugs upfront before the code is run. In the future, our external customers will need to be strong mode clean as well as Dart 2.0 evolves and ships. IMO It is better to turn this on now and encourage people to build better strong mode clean apps rather than letting bugs slip through to runtime *and* hitting them with a bunch of cleanup work in 6, 12, 18 months when Dart 2.0 makes this a requirement.

`flutter analyze` has been reporting strong mode violations up to this point... let's continue to do so.